.github/workflows/manylinux_2014.yml

```
@@ -0,0 +1,193 @@
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_22_04_x64_docker
+            ov_build/manylinux2014_x86_64
+          registry: 'openvinogithubactions.azurecr.io'
+          dockerfiles_root_dir: '.github/dockerfiles'
+          changed_components: ${{ needs.smart_ci.outputs.changed_components }}
+          
+  Build:
+    needs: [Docker]
+    timeout-minutes: 120
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores-32gb-manylinux
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_x64_docker }}
+      volumes:
+        - /mount:/mount
+      options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING -e DOCKER_CONFIG -v ${{ github.workspace }}:${{ github.workspace }}
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      OPENVINO_REPO: ${{ github.workspace }}/src
+      INSTALL_DIR: ${{ github.workspace }}/install/openvino
+      INSTALL_WHEELS_DIR: ${{ github.workspace }}/install/wheels
+      BUILD_DIR: ${{ github.workspace }}/build
+      DOCKER_CONFIG: "/mount/.docker"
+      CMAKE_CXX_COMPILER_LAUNCHER: sccache
+      CMAKE_C_COMPILER_LAUNCHER: sccache
+      SCCACHE_IGNORE_SERVER_IO_ERROR: 1
+      SCCACHE_SERVER_PORT: 35555
+      SCCACHE_CACHE_SIZE: 50G
+      SCCACHE_AZURE_KEY_PREFIX: manylinux_2014
+      
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: ${{ env.OPENVINO_REPO }}
+          submodules: 'true'
+
+      - name: System info
+        uses: ./src/.github/actions/system_info
+
+      - name: Create docker build cache
+        run: |
+          docker volume create ov_build_cache
+          
+      - name: Build OpenVINO
+        run: |
+          docker run --rm \
+            -v ${{ env.OPENVINO_REPO }}:/work/src \
+            -v ov_build_cache:/work/build \
+            -v ${{ env.INSTALL_DIR }}:/work/install \
+            -e SCCACHE_AZURE_BLOB_CONTAINER \
+            -e SCCACHE_AZURE_CONNECTION_STRING \
+            -e SCCACHE_SERVER_PORT \
+            -e SCCACHE_IGNORE_SERVER_IO_ERROR \
+            -e SCCACHE_CACHE_SIZE \
+            -e SCCACHE_AZURE_KEY_PREFIX \
+            -e CMAKE_CXX_COMPILER_LAUNCHER \
+            -e CMAKE_C_COMPILER_LAUNCHER \
+            -w /work/src \
+            ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
+            /bin/bash -c "
+              cmake -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DBUILD_SHARED_LIBS=ON -DENABLE_PYTHON=OFF -DENABLE_WHEEL=OFF -S /work/src -B /work/build &&
```

```suggestion
              cmake -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DENABLE_PYTHON=OFF -DENABLE_WHEEL=OFF -S /work/src -B /work/build &&
```
as `BUILD_SHARED_LIBS` is `ON` by default:
https://github.com/openvinotoolkit/openvino/blob/629fb81d520490ae7c4343614f5d614b69659b94/cmake/developer_package/features.cmake#L54-L56